Sources/Orbit/Support/Components/TimelineStep.swift

```diff
@@ -52,49 +57,82 @@ public struct TimelineStep<Header: View, Content: View>: View {
         .anchorPreference(key: TimelineStepPreferenceKey.self, value: .bounds) {
             [TimelineStepPreference(bounds: $0, style: style)]
         }
+        .onAppear { shouldAnimate = !reduceMotion }
```

Comment:
As most usages are in ternary operators, we might not need the additional @State property and `reduceMotion` can be used directly.

Reply:
in those ternary operators it is crucial that the variable changes its state from `false` to `true` in order to launch animation. If we use `reduceMotion` directly, the animation does not start

Reply:
I got it now. Then maybe lets rename it to `animationLoopTrigger` or similar, because this sounds like a simple property of whether the component should animate or not.

Reply:
renamed